src/history/hs_conn.c

```diff
@@ -133,22 +133,29 @@ int
 __wt_hs_open(WT_SESSION_IMPL *session, const char **cfg)
```

Comment:
Shall we just pass in a conn instead of session here? It is confusing that we still pass the session here.

Reply:
My 2 cents as I faced this before and asked the same question in another PR, the answer I got: most of our APIs start with a `WT_SESSION_IMPL *session`, we keep it even when unused for consistency.

Reply:
Yes, that's correct.